website/docs/docs/dbt-versions/upgrade-dbt-version-in-cloud.md

```diff
@@ -77,14 +77,14 @@ For more on version support and future releases, see [Understanding <Constant na
 ### dbt Fusion engine <Lifecycle status="beta" />
 
 dbt Labs has introduced the new [<Constant name="fusion_engine" />](/docs/fusion/about-fusion), a ground-up rebuild of dbt. This is currently in beta on the dbt platform. Eligible customers can update environments to Fusion using the same workflows as v1.x, but there are a few things to keep in mind:
-- **To gain access to the Fusion Latest release track, you must reach out to your dbt Labs account team to request it. Week by week we'll expand the beta cohort based on project eligibility, including Starter plans**. Once we transition from Beta to Preview, all users will see it as an option for their environments, projects, jobs, etc.
+- **To gain access to the Latest Fusion release track, you must reach out to your dbt Labs account team to request it. Week by week we'll expand the beta cohort based on project eligibility, including Starter plans**. Once we transition from Beta to Preview, all users will see it as an option for their environments, projects, jobs, etc.
```

Comment:
Not sure why we bolded this in the first place based on our style guide. Suggest we either make it normal text or transition it to a callout. 
```suggestion
- To gain access to the **Latest Fusion** release track, you must contact your dbt Labs account team to request it. Week by week, we'll expand the beta cohort based on project eligibility, including the Starter plan. Once we transition from Beta to Preview, all accounts will see it as an option for their environments, projects, jobs, etc.
```